src/main/scala/nl.knaw.dans.easy.stage/dataset/AMD.scala

```diff
@@ -52,7 +57,13 @@ object AMD extends DebugEnhancedLogging {
         <assigneeId>NOT_ASSIGNED</assigneeId>
         <wfs:workflow xmlns:wfs="http://easy.dans.knaw.nl/easy/workflow/">
           <id>dataset</id>
-          <remarks></remarks>
+          <remarks>
+            <remark>
+              <text>{ remarksContent }</text>
+              <remarkerId>{ s"easy-stage-dataset_$stageDatasetVersion" }</remarkerId>
```

Comment:
could we make this similar to https://github.com/DANS-KNAW/easy-stage-dataset/blob/91faffc4f66cd2eaab97c239fde4485376e5b9c3/src/main/scala/nl.knaw.dans.easy.stage/CommandLineOptions.scala#L36

```suggestion
              <remarkerId>{ s"easy-stage-dataset v$stageDatasetVersion" }</remarkerId>
```

Reply:
A space in an ID roused my suspicion. As far as I know it might even have been intended as a userID.

Reply:
At least for the steps the `remarkerId` was definitely intended as user id:
https://github.com/DANS-KNAW/easy-app/blob/b46a267421b7dc5443c39df995967690f25ff062/lib/easy-business/src/main/java/nl/knaw/dans/easy/domain/dataset/DatasetSpecification.java#L108